src/utils/user_registry.ts

```diff
@@ -61,6 +61,10 @@ export const blurUser = (
           return TE.left<ErrorResponses>(
             ResponseErrorInternal("Error calling PDV subsystem")
           );
+        default: 
+          return TE.left<ErrorResponses>(
```

Comment:
This will enforce the rule at compile time
```suggestion
          const _: never = res.status;
          return TE.left<ErrorResponses>(
```